apps/www/src/lib/registry/default/example/checkbox-disabled.svelte

```diff
@@ -7,7 +7,7 @@
 	<Checkbox id="terms" disabled />
 	<Label
 		for="terms2"
-		class="text-sm font-medium leading-none peer-disabled:cursor-not-allowed peer-disabled:opacity-70"
+		class="text-sm font-medium leading-none peer-disabled:cursor-not-allowed peer-disabled:opacity-70 peer-data-[disabled=true]:cursor-not-allowed peer-data-[disabled=true]:opacity-70"
```

Comment:
I think you should be able to do `peer-data-[disabled]:cursor-not-allowed`, etc., as if I remember correctly we don't set that property at all if it _isn't_ disabled.

Reply:
I just double checked and it looks like `data-disabled="false"` is added when not disabled.

![image](https://github.com/huntabyte/shadcn-svelte/assets/786592/6566e9ad-b8f5-48d8-9291-4c8babfdf479)

Reply:
Interesting, that shouldn't be the case but will work it out on the Melt side. In the meantime, this looks good to merge, thanks for this!